website/docs/en/contribute/development/profiling.md

```diff
@@ -67,6 +67,19 @@ npm install -g speedscope
 speedscope CPU.20230522.154658.14577.0.001.cpuprofile
 ```
 
+### Rsdoctor Timeline
+
+If we want to analyze the time consumption of loaders and plugins or the compilation behavior of loaders, we can use Rsdoctor to view:
```

Comment:
```suggestion
If you need to analyze the time cost of loaders and plugins, or the compilation behavior of loaders, you can use Rsdoctor to view:
```

Reply:
This is because is always use 'we' in this document.